graphql/resolvers.go

```diff
@@ -1392,14 +1392,6 @@ func (r *queryResolver) TaskTests(ctx context.Context, taskID string, execution
 			if err = apiTest.BuildFromService(&t); err != nil {
 				return nil, InternalServerError.Send(ctx, err.Error())
 			}
-			if err = util.CheckURL(utility.FromStringPtr(apiTest.Logs.HTMLDisplayURL)); apiTest.Logs.HTMLDisplayURL != nil && err != nil {
```

Comment:
These fields no longer exist. I removed them since they duplicated `URL` and `URLRaw`. They were there because sometimes we needed to prepend the api URL (`https://evergreen.mongodb.com`).